code/game/machinery/iv_drip.dm

```diff
@@ -69,7 +69,7 @@
 
 /obj/machinery/iv_drip/MouseDrop(mob/living/target)
 	. = ..()
-	if(!ishuman(usr) || !usr.canUseTopic(src, BE_CLOSE) || !isliving(target))
+	if( ( !ishuman(usr) && !iscyborg(usr) ) || !usr.canUseTopic(src, BE_CLOSE) || !isliving(target))
```

Comment:
Just remove the ishuman check instead, canUseTopic should be doing the work for us.